modules/distribution/resources/migration-2.6.0_to_3.0.0/versioning-disable-scripts/h2.sql

```diff
@@ -0,0 +1,40 @@
+-- Update the REG_PATH_ID column mapped with the REG_RESOURCE table --
+
```

Comment:
We will need to add a section to migration document? Do we have a draft version of it otherwise we might miss it.  Again customers in existing version may need this if they decided to migrate. We need to carefully handle this and properly documented.

Reply:
[1] is the draft version which does not contains complete steps as we need to finalize on IS component migration. Further, I have created [2], [3] to be documented which are identified during testing.
[1] https://docs.google.com/document/d/1QVDL3sUFxQxMNNqPF-WHe7FhM6SR0gvVLzdECQPYqz8/edit?usp=sharing
[2] https://github.com/wso2/product-apim/issues/5183
[3] https://github.com/wso2/product-apim/issues/5182